code/game/machinery/harvester.dm

```diff
@@ -94,22 +97,15 @@
 	update_icon()
 	if(!harvesting || state_open || !powered() || !occupant || !iscarbon(occupant))
 		return
-	playsound(src, 'sound/machines/juicer.ogg', 20, 1)
+	playsound(src, 'sound/machines/beeplaser.ogg', 20, 1)
 	var/mob/living/carbon/C = occupant
 	if(!LAZYLEN(operation_order)) //The list is empty, so we're done here
 		end_harvesting()
 		return
 	var/turf/target
```

Comment:
```suggestion
	var/turf/target = get_turf(src)
```
This will set the target to the turf that the harvester is currently located on.